py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -805,6 +812,442 @@ def test_expected_flux(self):
         expected_flux.extract_deltas(forest)
         self.assertTrue(all(forest.deltas == np.zeros_like(forest1.flux)))
 
+    def test_true_continuum(self):
+        """Test constructor for class TrueContinuum
+        
+        Load a TrueContinuum instance.
+        """
+        config = ConfigParser()
+        config.read_dict({
+                "expected flux": {
+                     "input directory": f"{THIS_DIR}/data",
+                     "iter out prefix": f"{THIS_DIR}/results/iter_out_prefix",
+                     "out dir": f"{THIS_DIR}/results"
+                }})
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        # this should raise an error as iter out prefix should not have a folder
+        with self.assertRaises(ExpectedFluxError):
+            expected_flux = TrueContinuum(config["expected flux"])
+
+        config = ConfigParser()
+        config.read_dict(
+            {"expected flux": {
+                "input directory": f"{THIS_DIR}/data",
+                "iter out prefix": "iter_out_prefix",
+                "out dir": f"{THIS_DIR}/results/",
+            }})
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        # this should also raise an error as Forest variables are not defined
+        with self.assertRaises(ExpectedFluxError):
+            expected_flux = TrueContinuum(config["expected flux"])
+
+        # setup Forest variables; case: logarithmic wavelength solution
+        setup_forest("log", rebin=3)
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        self.assertTrue(isinstance(expected_flux.get_var_lss, interp1d))
+
+        # setup Forest variables; case: linear wavelength solution
+        reset_forest()
+        setup_forest("lin", pixel_step=2.4)
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        self.assertTrue(isinstance(expected_flux.get_var_lss, interp1d))
+
+        # Assert invalid binning raises ExpectedFluxError
+        with self.assertRaises(ExpectedFluxError):
+            setup_forest("lin", pixel_step=0.4)
+            expected_flux = TrueContinuum(config["expected flux"])
+
+    def test_true_continuum_read_raw_statistics(self):
+        """Test reading raw statistics files"""
+        # setup Forest variables; case: linear wavelength solution
+        setup_forest("lin", pixel_step=2.4)
+
+        data_dir = Path(__file__).parent / "data"        
```

Comment:
use variable `THIS_DIR` to access the directories. Not sure where the path is, but you could define another variable similar to `THIS_DIR` at the top that points to the right place